sdk/webpubsub/Azure.ResourceManager.WebPubSub/tests/SessionRecords/PrivateEndpointConnectionTests/CheckIfExist.json

```diff
@@ -62,7 +62,7 @@
             "categories": [
               {
                 "name": "category-01",
-                "enabled": "true"
+                "enabled": "True"
```

Comment:
Service change the return value?

Reply:
According to ApiView comments, I added a new custom field here for converting bool and string. But I think I missed a thing, the result of `true.ToString()` is `"True"`, so in order for the results to be consistent, it should be added the `ToLower()` function. 
I don't know if the CultureInfo uses en-us,  please review these [customization files](https://github.com/Azure/azure-sdk-for-net/pull/29801/files#diff-a876b33fb184f3d0498cf5e3d1a98569fa3e4725309a0cedad0da8219dd5ab3eR50) again thanks